lib/models/patch/multi-file-patch.js

```diff
@@ -321,6 +339,13 @@ export default class MultiFilePatch {
     return false;
   }
 
+  getBufferRowForDiffPosition = (fileName, diffRow) => {
+    // TODO verify that this works on Windows
```

Comment:
🚨 Oh right, I still need to do this. I think dotcom always gives us `fileName` with `/` separators, but we normalize them to `\` before display.

Reply:
Spoiler alert: they were not handled correctly. It was the worst of both worlds: we were showing dotcom diffs with `/` separators, but _also_ throwing an exception here when trying to load comments.

<img width="403" alt="paths-not-fixed-up" src="https://user-images.githubusercontent.com/17565/51040237-f01fe800-1584-11e9-8dc7-969ede6d97a0.PNG">

I've fixed the path normalization so that the "Files" tab will (a) display paths with native path separators and (b) correctly compute comment offsets with the normalized paths.

<img width="358" alt="properly-normalized" src="https://user-images.githubusercontent.com/17565/51040298-18a7e200-1585-11e9-82b0-25ea0e1090f3.PNG">

<img width="420" alt="yo-dawg" src="https://user-images.githubusercontent.com/17565/51040295-15acf180-1585-11e9-97db-cb035d7344db.PNG">


![image](https://user-images.githubusercontent.com/17565/51040334-32492980-1585-11e9-801b-930af17146a5.png)